### PR TITLE
test: remove wrangler helpers

### PR DESCRIPTION
**Pull request by danielgent**
A momentous day :joy:

### CODE REVIEW

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -30,4 +30,3 @@ jobs:
       - run: yarn build
       - run: yarn type-check
       - run: yarn test
-      - run: yarn integration
```
Comment (danielgent) [inline .github/workflows/unit-tests.yml:33]: You mentioned Markus running one happy-path wrangler test...  I'm not sure if we'd miss something by only running hono/testing

File: test/integration/mockOauth2Client.ts
Comment (danielgent) [inline test/integration/mockOauth2Client.ts:1]: this kind of thing is so easy to do in VS code with relative imports. It just asks if I want to update, when I say yes, all the paths in this and all other files are updated

I hate aliases and absolute imports :joy:   _"but they make the code look clean"_ and then nothing else works

I also changed my VS code to use relative imports when doing autoimports

**Review by markusahlstrand — APPROVED** [review]
Looks good!

File: .github/workflows/unit-tests.yml (hunk 1 above)
Comment (markusahlstrand) [reply .github/workflows/unit-tests.yml:33]: I think we can run some e2e-tests later with playwright

**Pull request merged**